lms/djangoapps/teams/csv.py

```diff
@@ -289,11 +290,13 @@ def add_user_to_team(self, user_row):
             if not team_name:
                 continue
             if (team_name, teamset_id) not in self.existing_course_teams:
+                protected = user_row['mode'] in ORGANIZATION_PROTECTED_MODES
```

Comment:
We don't do any normalizing of data in `user_row` before we get to this point, so you might have e.g. "Masters" in "masters", which would return `False` when the user intent is probably something different.  So my minor suggestion is to either use `user_row['mode'].lower()` here, or do it higher up in the call chain, so that other helper methods don't have to do the same thing.

Reply:
Updated this check to use the built-in enum and user enrollment status